pkg/kubernetes/node/v1alpha1/node_test.go

```diff
@@ -0,0 +1,194 @@
+package v1alpha1
+
+import (
+	"testing"
+
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit : `corev1` as we are using the same alias in other places too ?

Reply:
Done